### PR TITLE
feat: add rsamtrig-config app

### PR DESCRIPTION
**Pull request by ozym**
This application is aimed at building and maintaining the sc configuration for the rsamtrig application

### CODE REVIEW

File: cmd/rsamtrig-config/main.go [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+	"github.com/GeoNet/delta"
+	"github.com/GeoNet/delta/meta"
+)
+
+type Settings struct {
+	base     string  // optional delta base directory
+	level    float64 // default rsam level
+	output   string  // output caps key directory
+	stations string  // list of stations to exclude
```
Comment (wilsonjord) [inline cmd/rsamtrig-config/main.go:20]: issue: I can't relate this comment to what is going on in the code

Comment (ozym) [reply cmd/rsamtrig-config/main.go:20]: agreed, this is a copy/paste error without updating the comment.

File: cmd/rsamtrig-config/main.go [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+	// process each station, expected label as "STN:LEVEL", if the level is
+	// missing then a default value will be used.
+	check := make(map[string]float64)
+	for _, s := range strings.Split(settings.stations, ",") {
+		if s = strings.TrimSpace(strings.ToUpper(s)); s == "" {
+			continue
+		}
+		switch {
+		case strings.Contains(s, ":"):
+			parts := strings.Fields(strings.ReplaceAll(s, ":", " "))
```
[the remark this answers is not in the document]
Comment (ozym) [reply cmd/rsamtrig-config/main.go]: This was a lazy first attempt which worked until it hit the next requirement, I had updated the code to use a regexp but I can't see why I didn't update the PR.  I've pushed the updates now, lives in a `config.go` file.

**Review by wilsonjord — APPROVED** [review]
seems fine. tests pass.

**Pull request merged**